Assembly-CSharp/Assembly-CSharp.csproj

```diff
@@ -27,6 +27,11 @@
 
     <PropertyGroup>
         <OutputDir>$(SolutionDir)/OutputFinal</OutputDir>
+        <!-- todo macOS -->
+        <GamePath Condition="'$(OS)' == 'Windows_NT'">C:/Program Files (x86)/Steam/steamapps/common/Hollow Knight</GamePath>
```

Comment:
I'd ideally give an env variable for this because these paths aren't universal (external drive steam library, flatpak, etc). Or Directory.Build.props. Can still default if it's unset but then people with less standard paths don't have to mess with the checked in csproj